src/Traits/UsersTrait.php

```diff
@@ -6,6 +6,8 @@
 use Illuminate\Database\Eloquent\Builder;
 use Junges\ACL\Exceptions\GroupDoesNotExistException;
 use Junges\ACL\Exceptions\PermissionDoesNotExistException;
+use Junges\ACL\Http\Models\Permission;
```

Comment:
I think it is not a good idea to use the Permission and Group model here, since the user can extend and replace this model with a custom one. This is why we have the `$model` within some functions, retrieving the model pathfrom a config file.
See [extending and replacing models][1] on readme file for more info.

[1]: https://github.com/jungessolutions/laravel-acl#extending-and-replacing-models

Reply:
Yes, I see. Totally forgot about that :sweat_smile: I'll make some changes so that both checks pass, and I'll grab Permission & Group classes from the configs, just as you did before :slightly_smiling_face: 
Keep in mind that this is my first time submitting a PR, so it may take me a few minutes to figure out how to actually make and commit those changes to the PR :smile:

Reply:
As soon as you push your changes, it will automatically be added to the PR, no worries about that. Congratulations for your first PR!